Formula/gdal.rb

```diff
@@ -92,6 +92,7 @@ def install
       "--with-pg=yes",
       "--with-png=#{Formula["libpng"].opt_prefix}",
       "--with-spatialite=#{Formula["libspatialite"].opt_prefix}",
+      "--with-pcre2=yes",
```

Comment:
No `opt_prefix`?

Reply:
`brew linkage` was already happy with the current change. I'm not sure if this takes such an argument